### ISSUE TITLE
Stuck keys issue

### ISSUE THREAD

**Issue by happenslol** [phamfoo/form-zero#1]
Hey, first of all thanks so much for making this keyboard open source!

I've built my own version of this, and I've been having an issue that I can't really explain, so I'm trying to find anyone who might have an idea on what it could be. I realize this might not even be an issue with this keyboard specifically, but I haven't ruled that out yet so I figured asking here wouldn't hurt.

From time to time (I'd say every 1-2 hours), a single key on the keyboard gets stuck and repeats for a few seconds, as if held down. It's definitely not a switch issue, since it's happened with 2 different form-zeros I've built, with different switches, PCBs and controllers. Could it be an issue with the driver or the PCB wiring?

Please feel free to close this if you're not looking to provide any support here. I'm just trying all avenues since I can't figure out what's causing it, and it's slowly driving me insane 😅

**Comment by phamfoo** [comment]
The PCB is just a simple matrix connected to the GPIO pins of the RP2040 Zero, so I don't think that's the issue. Could be a firmware issue, have you tried building the firmware yourself with the latest version of qmk?

**Comment by happenslol** [comment]
> The PCB is just a simple matrix connected to the GPIO pins of the RP2040 Zero, so I don't think that's the issue.

Yeah, I figured as much. I would also expect very different behavior if it was something like a damaged trace.

> Could be a firmware issue, have you tried building the firmware yourself with the latest version of qmk?

Yeah, I'm on the latest qmk version, and I'm pretty much always compiling the firmware myself since I have custom keymaps.

My prime suspect right now is the USB hub I'm using. Still weird that it's only happening on windows as far as I an tell. Thanks anyways!

**Comment by phamfoo** [comment]
Doesn't look like something I can help with unfortunately. I'm leaving this open in case anyone finds a solution.

**Comment by happenslol** [comment]
Thanks for your tips in any case! I'll comment if it turns out that it has anything to to with the board (or in case it doesn't 😅)
